packages/world-modules/src/modules/erc20-puppet/ERC20System.sol

```diff
@@ -52,7 +51,7 @@ contract ERC20System is System, IERC20Mintable, PuppetMaster {
    * {IERC20-balanceOf} and {IERC20-transfer}.
    */
   function decimals() public view virtual returns (uint8) {
-    return Metadata.getDecimals(_metadataTableId(_namespace()));
+    return ERC20Metadata.getDecimals(_metadataTableId(_namespace()));
```

Comment:
I generally agree, but in this case i'd prefer to not do the refactor for time reasons. We could merge the erc721 PR into the erc20 PR instead of merging both into main individually